src/main/java/com/example/braveCoward/dto/team/AddMemberResponse.java

```diff
@@ -0,0 +1,3 @@
+package com.example.braveCoward.dto.team;
+
+public record AddMemberResponse(Long memberId, String role, String position, Long teamId, Long userId) {}
```

Comment:
```suggestion
public record AddMemberResponse(
    Long memberId, 
    String role,
    String position,
    Long teamId, 
    Long userId
)  {
}
```
이런식으로 개행하면 가독성이 좀 더 좋을것 같아요!

Reply:
넵 반영했습니다!